libs/@guardian/source-react-components-development-kitchen/src/footer-with-contents/FooterLinks.tsx

```diff
@@ -64,46 +76,68 @@ export const FooterLinks = ({
 	forceColumns = false,
 }: FooterLinksProps): EmotionJSX.Element => {
 	const useColumns = links.length > 3 || forceColumns;
+	const isFooterLink = (
+		link: FooterLink | FooterButton,
+	): link is FooterLink => {
+		return (link as FooterLink).href !== undefined;
```

Comment:
I _think_ you could use something like `href in link` without using type coercions.

e.g.
```ts
type A = { href: string, text: string }
type B = { text: string }

type C = A | B

function isA(c: C): c is A {
    return 'href' in c
}


const b = { text: 'Yo!' }
const bIsAnA = isA(b);
```